### PR TITLE
Improvement and bug fixed

### PR DESCRIPTION
**Pull request by zapion**

### CODE REVIEW

File: mtbf_job_runner.py [hunk 1]
```diff
@@ -80,16 +83,23 @@ def add_7mobile_action(self):
...
         # Make sure it's in fastboot mode, TODO: leverage all fastboot command in one task function
         if self.adb_test():
-            os.system("adb reboot boot-loader")
-        mem_str = str(memory)
-        os.system("fastboot setvar mem " + mem_str)
-        # Preventing from async timing of fastboot
-        time.sleep(5)
-        os.system("fastboot reboot")
+            os.system("adb reboot bootloader")
+            memory = 319
```
Comment (ypwalter) [inline mtbf_job_runner.py]: changed to be no other options but 319?

File: mtbf_job_runner.py [hunk 2]
```diff
@@ -80,16 +83,23 @@ def add_7mobile_action(self):
...
+            mem_str = str(memory)
+            os.system("fastboot oem mem " + mem_str)
+            # Preventing from async timing of fastboot
+            time.sleep(5)
```
Comment (ypwalter) [inline mtbf_job_runner.py]: With ANDROID_SERIAL set, you can actually go without time.sleep(5). fastboot will automatically wait for the device to come up.

File: task.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+"add_7mobile_action": true,
```
Comment (ypwalter) [inline task.json]: interesting when set 7mobile true but change memory false

Comment (ypwalter) [inline task.json]: Normal user should have both default false? (Partner)

File: task.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+"change_memory": false,
+"full_flash": true,
+"shallow_flash": false,
+"enable_certified_apps_debug": false
```
Comment (ypwalter) [inline task.json]: Suggest default true

File: mtbf_job_runner.py (hunk 1 above)
Comment (zapion) [inline mtbf_job_runner.py]: This one is a workaround;  action runner currently not able to accept parameters.
Will fix later in combo-runner

File: mtbf_job_runner.py (hunk 2 above)
Comment (zapion) [inline mtbf_job_runner.py]: ok, will remove it.

File: task.json (hunk 3 above)
Comment (zapion) [inline task.json]: For default value, I think they should be all turned off as you mentioned

**Comment by ypwalter** [comment]
I think it is fine now. r+

**Pull request merged**